tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_api.py

```diff
@@ -85,6 +85,47 @@ def test_save_aliases_after_logging_artifact(user, wandb_init):
     assert "hello" in aliases
 
 
```

Comment:
Also, would be good to have a test when trying to add "invalid" tags, things like characters that are not allowed, duplicate tags, removing a tag that doesn't exist, adding a tag when the same already exists.

Reply:
Oh yeah, good call -- agreed as well!